PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -913,3 +913,21 @@ bool matchInList(const TCHAR *fileName, const std::vector<generic_string> & patt
 	return false;
 }
 
+std::wstring GetLastErrorAsString(DWORD errorCode)

```

Comment:
I understand you just moved this function from PluginsManager.cpp but since everything else uses `generic_string`s personally I'd suggest going ahead and converting this function to use it. (I'm assuming this would be an easy change).